t/ssl_session_resumption.t

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/perl
+
+use warnings;
+use Test::More;
+use FindBin qw($Bin);
+use lib "$Bin/lib";
+use MemcachedTest;
+
+if (!enabled_tls_testing()) {
+    plan skip_all => 'SSL testing is not enabled';
+    exit 0;
+}
+
+my $server;
+my $sock;
+my $stats;
+
+my $session_cache = IO::Socket::SSL::Session_Cache->new(1);
```

Comment:
Think this needs to wrap with an eval as well, or it will fail to compile on machines missing SSL perl modules.

sorry this is so complicated :/ I might just do this myself if you can't soon. Need to spin up a VM missing the modules or something.

Reply:
No worries! Thanks for the review, I've updated to wrap this in eval as well. Please feel free to push additional commits to this branch as needed.

I was able to run this in a Docker container without the SSL Perl module installed and it failed properly without crashing:

```
1..0 # SKIP SSL testing is not enabled
```